Encrypted Media Extensions is not supported on OpenBSD. Apparently this also
means we can not pass --disable-eme to configure so comment that out.
Also see
https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/42501

Index: mozconfig-linux-x86_64
--- mozconfig-linux-x86_64.orig
+++ mozconfig-linux-x86_64
@@ -8,8 +8,5 @@ ac_add_options --disable-install-strip
 
 ac_add_options --enable-default-toolkit=cairo-gtk3
 
-# Let's make sure no preference is enabling either Adobe's or Google's CDM.
-ac_add_options --disable-eme
-
 # tor-browser#44067: geckodriver has become a host-only tool
 ac_add_options --enable-geckodriver
